applications/API/tests/test_endpoints.py

```
@@ -0,0 +1,287 @@
+
+    return OpenView
+
+
+def canon_cat(value):
+    # Some endpoints return key ("P"), others return name ("Poor"). Accept both.
+    return {"P": "Poor"}.get(value, value)
+
+
+class EndpointTests(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
```

Ruff (0.12.2)

Missing return type annotation for classmethod `setUpTestData`

Add return type annotation: `None`

(ANN206)